torch/_dynamo/graph_break_hints.py

```diff
@@ -0,0 +1,21 @@
+USER_ERROR = [
```

Comment:
I like the categories. It's just a bit hard to review labeling the error message with categories because I have nits on what the error messages are saying (in the previous PR)